src/content/en/updates/2021/02/css-in-js.md

```diff
@@ -0,0 +1,149 @@
+
+<<../../_shared/devtools-research.md>>
+
+This article talks about CSS-in-JS support in DevTools that landed since Chrome 85 and, in general, what we mean by CSS-in-JS and how it’s different from regular CSS that has been supported by DevTools for a long time.
+
+## What is CSS-in-JS?
+The definition of CSS-in-JS is rather vague. In a broad sense, it’s an approach for managing CSS code using JavaScript. For example, it could mean that the CSS content is defined using JavaScript and the final CSS output is generated on the fly by the app. 
+
+In the context of DevTools, CSS-in-JS means that the CSS content is injected into the page using [CSSOM APIs](/web/updates/2018/03/cssom). Regular CSS is injected using `<style>` or `<link>` elements, and it has a static source (e.g. a DOM node or a network resource). In contrast, CSS-in-JS often does not have a static source. A special case here is that the content of a `<style>` element can be updated using CSSOM API, causing the source to become out of sync with the actual CSS stylesheet.
+
+If you use any CSS-in-JS library (e.g. [styled-component](https://github.com/styled-components/styled-components), [Emotion](https://emotion.sh/), [JSS](https://cssinjs.org/)),  the library might inject styles using CSSOM APIs under the hood depending on the mode of development and the browser. 
+
+Let’s look at some examples on how you can inject a stylesheet using CSSOM API similar to what CSS-in-JS libraries are doing.
+
+```js
+// Insert new rule to an existing CSS stylesheet
+const element = document.querySelector('style');
+const stylesheet = element.sheet;
+stylesheet.replaceSync('.some { color: blue; }');
+stylesheet.insertRule('.some { color: green; }'); 
+```
+
+You can [create a completely new stylesheet](/web/updates/2019/02/constructable-stylesheets) as well:
+
+```js
+// Create a completely new stylesheet
+const sheet = new CSSStyleSheet();
+stylesheet.replaceSync('.some { color: blue; }');
+stylesheet.insertRule('.some { color: green; }'); 
+
+// Apply constructed stylesheet to the document
+document.adoptedStyleSheets = [...document.adoptedStyleSheets, sheet];
+```
+
+## CSS support in DevTools
+In DevTools, the most commonly used feature when dealing with CSS is the **Styles** pane. In the **Styles** pane, you can view what rules apply to a particular element and you can edit the rules and see the changes on the page in realtime. 
+
+<video src="/web/updates/images/2021/02/css-in-js/css-edit.mp4" alt="Edit CSS" style="max-width:100%;" autoplay loop muted></video>
+
+Before last year, the support for CSS rules modified using CSSOM APIs was rather limited: **you could only see the applied rules but could not edit them.** The main goal we had last year was to allow editing of CSS-in-JS rules using the Styles pane. Sometimes we also call CSS-in-JS styles [“constructed”](/web/updates/2019/02/constructable-stylesheets) to indicate that they were constructed using Web APIs.
```

Comment:
replace "the support" by "tooling support". And "before last year" by "a year ago".